diff --git a/fixtures/spa/atomic-crm/src/components/atomic-crm/layout/Header.tsx b/fixtures/spa/atomic-crm/src/components/atomic-crm/layout/Header.tsx
index 2e8f983..fd1c7e9 100644
--- a/fixtures/spa/atomic-crm/src/components/atomic-crm/layout/Header.tsx
+++ b/fixtures/spa/atomic-crm/src/components/atomic-crm/layout/Header.tsx
@@ -51,7 +51,7 @@ const Header = () => {
                 <h1 className="text-xl font-semibold">{title}</h1>
               </Link>
               <div>
-                <nav className="flex">
+                <nav className="flex gap-1">
                   <NavigationTab
                     label={translate("ra.page.dashboard")}
                     to="/"
