diff --git a/fixtures/spa/atomic-crm/src/components/atomic-crm/tasks/Task.tsx b/fixtures/spa/atomic-crm/src/components/atomic-crm/tasks/Task.tsx
index a4e856c..02d5036 100644
--- a/fixtures/spa/atomic-crm/src/components/atomic-crm/tasks/Task.tsx
+++ b/fixtures/spa/atomic-crm/src/components/atomic-crm/tasks/Task.tsx
@@ -155,7 +155,6 @@ export const Task = ({
               variant="ghost"
               size="icon"
               className="h-5 pr-0! size-8 cursor-pointer"
-              aria-label={translate("resources.tasks.actions.title")}
             >
               <MoreVertical className="size-5 md:size-4" />
             </Button>
