diff --git a/fixtures/spa/atomic-crm/src/components/admin/notification.tsx b/fixtures/spa/atomic-crm/src/components/admin/notification.tsx
index 4151e3b..66734de 100644
--- a/fixtures/spa/atomic-crm/src/components/admin/notification.tsx
+++ b/fixtures/spa/atomic-crm/src/components/admin/notification.tsx
@@ -108,7 +108,6 @@ export const Notification = (props: ToasterProps) => {
   return (
     <CloseNotificationContext.Provider value={handleRequestClose}>
       <Toaster
-        richColors
         theme={theme}
         closeButton
         position="bottom-center"
