diff --git a/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactInputs.tsx b/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactInputs.tsx
index c56809a..4fd1d37 100644
--- a/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactInputs.tsx
+++ b/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactInputs.tsx
@@ -73,7 +73,7 @@ const ContactIdentityInputs = () => {
         defaultValue={contactGender[0].value}
       />
       <TextInput source="first_name" validate={required()} helperText={false} />
-      <TextInput source="last_name" validate={required()} helperText={false} />
+      <TextInput source="last_name" label={false} validate={required()} helperText={false} />
     </div>
   );
 };
