diff --git a/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactShow.tsx b/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactShow.tsx
index 6cc24ee..ff267d7 100644
--- a/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactShow.tsx
+++ b/fixtures/spa/atomic-crm/src/components/atomic-crm/contacts/ContactShow.tsx
@@ -248,7 +248,7 @@ const ContactShowContent = () => {
             <div className="flex">
               <Avatar />
               <div className="ml-2 flex-1">
-                <h5 className="text-xl font-semibold">
+                <h5 className="text-2xl font-semibold">
                   <RecordRepresentation />
                 </h5>
                 <div className="inline-flex text-sm text-muted-foreground">
