diff --git a/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx b/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
index e45a2c2..5f75a14 100644
--- a/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
+++ b/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
@@ -31,6 +31,8 @@ import {
     useCreateSuggestionContext,
     EditActionsProps,
     CanAccess,
+    TabbedFormTabs,
+    TabbedFormTabsProps,
 } from 'react-admin';
 import {
     Box,
@@ -98,9 +100,23 @@ const categories = [
     { name: 'Lifestyle', id: 'lifestyle' },
 ];
 
+const StyledTabs = (props: TabbedFormTabsProps) => (
+    <TabbedFormTabs {...props}>
+        {React.Children.map(props.children, tab =>
+            React.isValidElement(tab)
+                ? React.cloneElement(tab, { 'aria-selected': false } as any)
+                : tab
+        )}
+    </TabbedFormTabs>
+);
+
 const PostEdit = () => (
     <Edit title={<PostTitle />} actions={<EditActions />}>
-        <TabbedForm defaultValues={{ average_note: 0 }} warnWhenUnsavedChanges>
+        <TabbedForm
+            defaultValues={{ average_note: 0 }}
+            warnWhenUnsavedChanges
+            tabs={<StyledTabs />}
+        >
             <TabbedForm.Tab label="post.form.summary">
                 <SanitizedBox
                     display="flex"
