diff --git a/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx b/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
index e45a2c2..e65bb11 100644
--- a/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
+++ b/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
@@ -204,6 +204,13 @@ const PostEdit = () => (
                     resettable
                     source="category"
                     choices={categories}
+                    SelectProps={{
+                        MenuProps: {
+                            onKeyDownCapture: event => {
+                                if (event.key === 'Escape') event.stopPropagation();
+                            },
+                        },
+                    }}
                 />
                 <NumberInput
                     source="average_note"
