diff --git a/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx b/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
index e45a2c2..a34ce96 100644
--- a/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
+++ b/fixtures/spa/react-admin-simple/src/posts/PostEdit.tsx
@@ -31,6 +31,7 @@ import {
     useCreateSuggestionContext,
     EditActionsProps,
     CanAccess,
+    TabbedFormTabs,
 } from 'react-admin';
 import {
     Box,
@@ -100,7 +101,11 @@ const categories = [
 
 const PostEdit = () => (
     <Edit title={<PostTitle />} actions={<EditActions />}>
-        <TabbedForm defaultValues={{ average_note: 0 }} warnWhenUnsavedChanges>
+        <TabbedForm
+            defaultValues={{ average_note: 0 }}
+            warnWhenUnsavedChanges
+            tabs={<TabbedFormTabs indicatorColor="secondary" />}
+        >
             <TabbedForm.Tab label="post.form.summary">
                 <SanitizedBox
                     display="flex"
