diff --git a/fixtures/spa/react-admin-simple/src/comments/PostReferenceInput.tsx b/fixtures/spa/react-admin-simple/src/comments/PostReferenceInput.tsx
index 35e02c4..dcf250c 100644
--- a/fixtures/spa/react-admin-simple/src/comments/PostReferenceInput.tsx
+++ b/fixtures/spa/react-admin-simple/src/comments/PostReferenceInput.tsx
@@ -61,6 +61,7 @@ const PostReferenceInput = () => {
                     </Button>
                     <Dialog
                         data-testid="dialog-show-post"
+                        className="post-preview-dialog"
                         fullWidth
                         open={showPreviewDialog}
                         onClose={handleCloseShow}
