diff --git a/fixtures/spa/react-admin-simple/src/customRouteLayout.tsx b/fixtures/spa/react-admin-simple/src/customRouteLayout.tsx
index 63436a8..1cb40a9 100644
--- a/fixtures/spa/react-admin-simple/src/customRouteLayout.tsx
+++ b/fixtures/spa/react-admin-simple/src/customRouteLayout.tsx
@@ -16,7 +16,7 @@ const CustomRouteLayout = ({ title = 'Posts' }) => {
             <Title title="Example Admin" />
             <h1>{title}</h1>
             <p>
-                Found <span className="total">{total}</span> posts !
+                Found <span className="total">{total}</span> posts.
             </p>
             <DataTable
                 sort={sort}
